blogposts/structural-code-search-at-scale-from-your-browser.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Structural code search. At scale and from your browser.
+author: Rijnard van Tonder
+authorUrl: https://twitter.com/rvtond
+publishDate: 2020-01-24T10:00-07:00
+tags: [blog]
+slug: structural-code-search-at-scale-from-your-browser
+heroImage: https://TODO
+published: true
+---
+
+We're introducing a richer way to search code at Sourcegraph with structural
```

Comment:
It's not clear what is meant by *richer*? Alternatives could be "enhanced", "more specialized", "next generation".

Reply:
Richer in terms of expressivity. But "more expressive" isn't the right phrase. In general "more X" prompts 'more X than what?' and if X is regex then I'm not sure isn't an accurate description (the comparison is vague). Saying "new" might be the best alternative.

Reply:
I like new the most.